src/invidious.cr

```diff
@@ -191,7 +191,7 @@ if (CONFIG.use_pubsub_feeds.is_a?(Bool) && CONFIG.use_pubsub_feeds.as(Bool)) ||
   Invidious::Jobs.register Invidious::Jobs::SubscribeToFeedsJob.new(PG_DB, HMAC_KEY)
 end
 
-if CONFIG.popular_enabled
+if CONFIG.page_enabled?("popular")
   Invidious::Jobs.register Invidious::Jobs::PullPopularVideosJob.new(PG_DB)
 end
```

Comment:
(**Not a blocking review just a comment**): Huh it looks like the popular page will stay empty if an admin enables the popular feed through the preferences page rather than the config at start-up, since the job will never be started.

This isn't something that's caused by the PR but rather Invidious' current behavior. 

It could be fixed via a select statement and some global channels but it should probably be handled by some sort of global job scheduler instead 🤔